Bottom: 40b09380e4790e2c132b32ff1dad31fb86b883d8
Top:    89410db33463eaa9308b123ea77a72bca37bf0e0
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-09-15 16:26:03 +0800

MAINTAINERS: Stefan will not maintain network device layer

Talked with Stefan, he will not maintain network device layer.

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/MAINTAINERS b/MAINTAINERS
index 08f356a..4f73fb7 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -954,11 +954,10 @@ F: hmp-commands.hx
 T: git git://repo.or.cz/qemu/qmp-unstable.git queue/qmp
 
 Network device layer
-M: Stefan Hajnoczi <stefanha@redhat.com>
 M: Jason Wang <jasowang@redhat.com>
 S: Maintained
 F: net/
-T: git git://github.com/stefanha/qemu.git net
+T: git git://github.com/jasowang/qemu.git net
 
 Netmap network backend
 M: Luigi Rizzo <rizzo@iet.unipi.it>
